content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -22,33 +22,25 @@ This flow assumes that the Istio components are installed and upgraded in the
 `istio-system` namespace.
 
 {{< warning >}}
-Be sure to check out the [upgrade notice](/docs/setup/kubernetes/upgrade/notice) for a concise list of things you should know before
-upgrading your deployment to Istio 1.1.
+Be sure to check out the [upgrade notice](/docs/setup/kubernetes/upgrade/notice)
+for a concise list of things you should know before upgrading your deployment to Istio 1.2.
 {{< /warning >}}
 
+{{< tip >}}
+Istio does **NOT** support skip level upgrades.  Only upgrades from 1.1 to 1.2
+are supported. If you are on an older version, please upgrade to 1.1 first.
+{{< /tip >}}
+
 ## Upgrade steps
 
 [Download the new Istio release](/docs/setup/kubernetes/#downloading-the-release)
 and change directory to the new release directory.
 
 ### Control plane upgrade
 
-{{< warning >}}
-Helm has significant problems when upgrading CRDs using Tiller.
-We believe we have solved these with the introduction of the `istio-init` chart.
-However, because of the wide variety of Helm and Tiller versions used with previous Istio deployments,
-ranging from 2.7.2 to 2.12.2, we recommend an abundance of caution by
-backing up your custom resource data, before proceeding with the upgrade:
-
-{{< text bash >}}
-$ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | cut -f1-1 -d "." | \
-    xargs -n1 -I{} sh -c "kubectl get --all-namespaces -oyaml {}; echo ---" > $HOME/ISTIO_1_0_RESTORE_CRD_DATA.yaml
-{{< /text >}}
-
-{{< /warning >}}
-
-The Istio control plane components include: Citadel, Ingress gateway, Egress gateway, Pilot, Galley, Policy, Telemetry and
-Sidecar injector.  Choose one of the following two **mutually exclusive** options to update the control plane:
+The Istio control plane components include: Citadel, Ingress gateway, Egress gateway,
+Pilot, Galley, Policy, Telemetry and Sidecar injector.  Choose one of the following
```

Comment:
Do we want to list each control plane components?  This varies per profile.  The list here isn't correct for our default profile (should be without egress gateway but with Prometheus).

Reply:
```suggestion
```

Reply:
good point, hadn't considered this aspect of the document.  The upgrade process strongly depends on what the defaults are in `values.yaml` not the demo profile.

I'll deploy the default and add an info box (operators will need to turn those control plane components on which they had on prior).